documentation/api/guides/debugging.md

```diff
@@ -69,3 +69,12 @@ This is the guide to debugging the API using VS Code. This uses Microsoft's
    [Visual Code debugger's official instructions](https://code.visualstudio.com/docs/editor/debugging)
    to better understand how to use the debugger interface to accomplish tasks
    like setting breakpoints, watching variables etc.
+
+## Debugging Database Queries
+
+To log all database queries, add the following `env` variables to your configuration
```

Comment:
I'd suggest rewording this to be a bit more specific:

```suggestion
To log all database queries, add the following variables to your `api/.env` file:
```

Reply:
@AetherUnbound Thank you for the suggestion, I've update the content and pushed the changes.